packages/imodel-content-tree/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@bentley/imodel-content-tree",
```

Comment:
@calebmshafer , should this pkg also use the `-react` suffix we've been using for the others?

Reply:
Yes, good catch.  @grigasp can you add that to the name of the package?  It's a standard we've been enforcing within this repo.

Reply:
You'll have to update the CODEOWNERS, version-policies.json, rush.json, and readme. Maybe a few more spots too